Python/Product/RunElevated/app.manifest

```diff
@@ -0,0 +1,40 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
+  <assemblyIdentity version="1.0.0.0" name="MyApplication.app"/>
```

Comment:
From what I understand of the doc, this is required but only really matters when other assemblies need to specify a dependency on this assembly.
Given that, maybe we can just fix the name and major/minor version without worrying about build number?